subsys/bluetooth/Kconfig

```diff
@@ -117,61 +117,7 @@ config BT_LIM_ADV_TIMEOUT
 	  Appendix A (NORMATIVE): TIMERS AND CONSTANTS it's required to be no more
 	  than 180s.
 
-config BT_EXT_ADV
-	bool "Extended Advertising and Scanning support [EXPERIMENTAL]"
-	help
-	  Select this to enable Extended Advertising API support.
-	  This enables support for advertising with multiple advertising sets,
-	  extended advertising data, and advertising on LE Coded PHY.
-	  It enables support for receiving extended advertising data as a
-	  scanner, including support for advertising data over the LE coded PHY.
-	  It enables establishing connections over LE Coded PHY.
-
-if BT_EXT_ADV
-config BT_EXT_ADV_LEGACY_SUPPORT
-	bool "Support starting advertising through legacy commands"
-	help
-	  Select this to enable the use of the Legacy Advertising HCI commands.
-	  This option should be used where the capabilities of the controller
-	  is not known.
-	  If this option is not enabled the controller must support the extended
-	  advertising feature.
-
-config BT_EXT_ADV_MAX_ADV_SET
-	int "Maximum number of simultaneous advertising sets"
-	range 1 64
-	default 1
-	help
-	  Maximum number of simultaneous Bluetooth advertising sets
-	  supported.
-
-config BT_PER_ADV
-	bool "Periodic Advertising and Scanning support [EXPERIMENTAL]"
-	help
-	  Select this to enable Periodic Advertising API support. This allows
-	  the device to send advertising data periodically at deterministic
-	  intervals. Scanners can synchronize to the periodic advertisements
-	  to periodically get the data.
-
-config BT_PER_ADV_SYNC
-	bool "Periodic advertising sync support [EXPERIMENTAL]"
-	depends on BT_OBSERVER
-	help
-	  Select this to enable Periodic Advertising Sync API support.
-	  Syncing with a periodic advertiser allows the device to periodically
-	  and deterministic receive data from that device in a connectionless
-	  manner.
-
-if BT_PER_ADV_SYNC
-config BT_PER_ADV_SYNC_MAX
-	int "Maximum number of simultaneous periodic advertising syncs"
-	range 1 64
-	default 1
-	help
-	  Maximum number of simultaneous periodic advertising syncs supported.
-endif # BT_PER_ADV_SYNC
-
-endif # BT_EXT_ADV
+rsource "Kconfig.ext_adv"
```

Comment:
There are some `BT_PER_ADV` in what you moved to Kconfig.ext_adv, maybe just move the rest of the adv section there as well (`BT_LIM_ADV_TIMEOUT`) and call it Kconfig.adv?

Reply:
I was thinking about it. However `BT_PER_ADV` depends on `BT_EXT_ADV` but I like an idea to just move all related advertising configs to one file. Will do

Reply:
Done